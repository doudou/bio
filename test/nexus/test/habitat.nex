#NEXUS
[/home/cverna/.arb_tmp/tmp/gde19576_1 -- data title]

[Name: Test_1          Len:  1159  Check: 2338B739]
[Name: Test_2          Len:  1159  Check: 2338B739]
[Name: Test_3          Len:  1159  Check:  580DE0F]
[Name: Test_4          Len:  1159  Check: DFA31037]
[Name: Test_5          Len:  1159  Check: 2338B739]

begin data;
 dimensions ntax=5 nchar=20;
 format datatype=rna interleave=yes missing=. gap=-;
  matrix
 Test_1  GCGUGGAAUCUGCCUAGUAG
 Test_2  GCGUGGAAUCUGCCUAGUAG
 Test_3  GCGUGGAAUCUGCCUAGUAG
 Test_4  GCGUGGAAUCUGCCUAGUAG
 Test_5  GCGUGGAAUCUGCCUAGUAG
;
  end;
BEGIN TRAITS;
  Dimensions NTRAITS=2;
  Format labels=yes missing=? separator=Comma;
  TraitLabels hab0 hab1;
  Matrix
Test_1 1,0
Test_2 2,1
Test_3 0,1
Test_4 0,1
Test_5 1,0
;
END;
