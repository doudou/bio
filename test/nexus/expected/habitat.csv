name,hab0,hab1
Test_1,1,0
Test_2,2,1
Test_3,0,1
Test_4,0,1
Test_5,1,0
